includes/payments/functions.php

```diff
@@ -1241,8 +1242,9 @@ function give_donation_amount( $donation, $format_args = false ) {
 	 * @param string $formatted_amount Formatted/Un-formatted amount.
 	 * @param float  $amount           Donation amount.
 	 * @param int    $donation_id      Donation ID.
+	 * @param string $type             Donation amount type.
 	 */
-	return apply_filters( 'give_donation_amount', (string) $formatted_amount, $amount, $donation );
+	return apply_filters( 'give_donation_amount', (string) $formatted_amount, $amount, $donation, $type );
```

Comment:
Little change

return apply_filters( 'give_donation_amount', (string) $formatted_amount, $amount, $donation, $type );


It should be 
return (string) apply_filters( 'give_donation_amount', $formatted_amount, $amount, $donation, $type );